lib/app/screens/about.dart

```diff
@@ -236,7 +274,7 @@ final List<PersonRecord> teamMembers = [
     name: "Robert Ace Gonzales",
     role: "Backend Engineer",
     image: Image.asset('assets/images/robert.png', fit: BoxFit.cover),
-    extraInformation: const Text('goodbye, cruel world!'),
+    extraInformation: const Text('i <3 the internet'),
```

Comment:
You need to actually use the `RobertExtraInfo` widget you created here.

You can add the little text blurb you added here to the column in the widget you created.

```suggestion
    extraInformation: const RobertExtraInfo(),
```